packages/celotool/src/cmds/deploy/initial/verification-pool.ts

```diff
@@ -11,16 +11,10 @@ export const builder = {}
 export const handler = async (argv: InitialArgv) => {
   const vpConfig = await getVerificationPoolConfig(argv.celoEnv)
   const projName = fetchEnvOrFallback(envVar.TESTNET_PROJECT_NAME, 'celo-testnet')
-  const appSignature = fetchEnvOrFallback(envVar.SMS_RETRIEVER_HASH_CODE, '')
+  // const appSignature = fetchEnvOrFallback(envVar.SMS_RETRIEVER_HASH_CODE, '')
```

Comment:
We should just remove this now, right?

Reply:
We could but I'd rather not further grow the scope of this PR by touching verification pool code

Reply:
I'm open to removing the pool at this point, though I still think the verifier should stay for now